.travis.yml

```diff
@@ -138,4 +138,5 @@ script:
   - codecept run views_rest
   - codecept run views_integration
   - codecept run views_ui
+  #- codecept run acceptance
```

Comment:
Looks good! And with tests in place! Are we adding this commented for any reason in particular?

Reply:
Yeah, adding as commented because I need to figure out some TravisCI issues that I'm having with the tests. I'll be figuring that out soon-ish!

Reply:
Perfect, thanks!